docs/kubernetes.adoc

```diff
@@ -74,7 +74,12 @@ You can configure Kubernetes by using any of the next configuration properties i
 |===
 | Options                             | Type           | Env | Description                                                   
 | kubernetes.master                   | URL            | Any | The URL to the Kubernetes master                               
-| kubernetes.domain                   | String         | OSE | Domain to use for creating routes for services                 
+| name                                | String         | Any | Username to log in server
+| password                            | String         | Any | Password to log in server
```

Comment:
How about calling parameters `kubernetes.user.name` `kubernetes.user.password` and `kubernetes.user.token`? Any thoughts about this @bartoszmajsak

Reply:
Do we want to be specific here? Or the idea is to have generic concept of `user` and `pwd` to be applied depends on the target cluster/server/cloud provider (you name it) :)

Reply:
It might also be confusing, as now OpenShift is k8s, but not everyone knows that?